src/bridgepicassospec.js

```diff
@@ -39,7 +39,7 @@ export default function (element, layout, direction, isInteractable, ds) {
     const measureInfo = ds.field(field).raw();
     const formatter = ds.field(field).formatter();
     const useFormatter = shouldUseFormat(measureInfo);
-    const formatCell = (cell) => (cell.qNum === 'NaN' ? '-' : formatter(cell.qNum));
+    const formatCell = (cell) => (cell.qNum === 'NaN' ? '-' : formatter(cell.qText));
```

Comment:
This is not correct. formatter need to take in a number as parameter not a string.

Reply:
The problem is we use data.end.label for bar labels and this is wrong. We should use data.var.label for bar labels. So what you are doing here is totally not correct.
I suggest the fix https://github.com/AnalyticsEarth/variance-waterfall/pull/20